src/autocomplete/databases/yql/yql-autocomplete.ts

```diff
@@ -46,6 +47,33 @@ const tokenDictionary: TokenDictionary = {
     SELECT: YQLParser.SELECT,
 };
 
+function getUniqueTableSuggestions(suggestions: Table[] = []): Table[] {
```

Comment:
did you manage to figure out why we have duplicate table names anyway?
I remember handling that error, but don't remember why it originated 😄